### DISCUSSION TITLE
Night Mode on Arena, Compactor brightness, contrast and color setting

### DISCUSSION THREAD

**Discussion by Wazzted** [SmartlyDressedGames/Unturned-3.x-Community#4781] [Ideas]
Hi there,

I have been working on my Unturned Arena Map for quite a while now, and Me and my Pals who I play with had an idea to make the map dark so its harder..
First I was experimenting in Lightning tab and set dense brownish fog with full dark sky top mid and bottom and a little of ambient top mid and bot as well.. 
This made the "Day" in game pretty dark and kinda like from horror or something... 

Now I changed it to actual night using Bias and Fade setting somehow I managed to turn it... but the Time slider doent effect it at all.. The map looks absolutely as it should by my imagination altho there is one more issue..

The compactor brightness and contrast: As I was playing with my fellas, We had issue spot each other in between the Bright yellow compactor border, so I have been searching for options how can I set it and most likely the compactor is core game mechanic with some configurations that are adjustable and other that arent... I would like the brightness contrast and color to be changed to fit my map..

I dont know what to do as We are specialists of hard PvP action and We are fond of playing on Night setting with denser fog to prevent sniping..

**Comment by SDGNelson** [comment]
You're right that there aren't currently options to adjust the yellow color, sorry!

On the bright side, perhaps it provides an incentive not to stay near the border? If the border gives away your position then it's a good idea to stay further in to the arena.
